docs/_data/topnav_cn.yml

```diff
@@ -3,8 +3,8 @@
 topnav:
 - title: Topnav
   items:
-    - title: 新闻
-      url: /news
```

Comment:
why get dropped?

Reply:
because the english doc has removed this.
![image](https://user-images.githubusercontent.com/20113411/69288824-ab99ba00-0c35-11ea-98c3-893f166ad5bc.png)